setup-tools/generate-cluster-config.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -euo pipefail
+
+KEY_PATH=$1
+INSTANCE_IP=$2
+
+CONFIG_FILE="config.json"
+
+CLUSTER_NAME=$(jq -r '.ClusterName' $CONFIG_FILE)
+DEVICE_LIST=""
+
+LEN=$(jq '.Devices | length'  $CONFIG_FILE)
+for i in $(seq 0 $[LEN - 1])
+do
+    IP=$(jq -r '.Devices['$i'].IPAddress' $CONFIG_FILE)
+    DEVICE_LIST+="  - $IP\n"
```

Comment:
it can only help to modify some of the configurable fields, not a very smart script. User can always modify the cluster config manually. Will add same instructions in the readme.